packages/cargo/src/graph/index.ts

```diff
@@ -13,6 +13,7 @@ export function processProjectGraph(
 ): ProjectGraph {
 	let metadata = cp.execSync("cargo metadata --format-version=1", {
 		encoding: "utf8",
+		maxBuffer: 1208 * 1208 * 1208 * 16,
```

Comment:
Forgive my ignorance if there's something I'm missing, but 1208 seems like a strange/arbitrary number that makes it harder to do the math in your head. Why not `1024 * 1024 * 1024 * 32` for an even 32 MB?

```suggestion
		maxBuffer: 1024 * 1024 * 1024 * 32,
```

Reply:
ah yes i was very tired when i wrtoe it, that is correct